src/EventListener/FlashMessageListener.php

```diff
@@ -48,6 +48,7 @@ public function __construct($session, array $options = [])
         $this->session = $session;
 
         $resolver = new OptionsResolver();
+        $resolver->setDefined(['name', 'path', 'host', 'secure', 'enabled']);
         $resolver->setAllowedTypes('name', 'string');
         $resolver->setAllowedTypes('path', 'string');
         $resolver->setAllowedTypes('host', 'string');
```

Comment:
according to the default configuration, the host may also be `null`, actually. although with the code below, i am not sure that this will work out fine. line 91 about `if (isset($cookies[$this->options['host']][$this->options['path']][$this->options['name']])) {` will never work if host is null i think.

if you use the listener, can you check if that check makes sense and works as intended?